src/cli.rs

```diff
@@ -89,6 +91,13 @@ pub struct Cli {
     #[structopt(short = "s", long = "style", possible_values = &Theme::variants(), case_insensitive = true)]
     pub theme: Option<Theme>,
 
+    /// Proxy to use for a specific protocol. The value passed to this option should take the form of
+    /// <PROTOCOL>:<PROXY_URL>. You can specify proxies for multiple protocols by repeating this option.
+    /// Note that when this option is absent, the environment variables `http_proxy` and
+    /// `https_proxy` can be used to set proxies to use.
+    #[structopt(long, value_name = "<PROTOCOL>:<PROXY_URL>", number_of_values = 1)]
```

Comment:
This looks less noisy in the help text:
```suggestion
    #[structopt(long, value_name = "PROTOCOL:PROXY_URL", number_of_values = 1)]
```

Reply:
Agreed :)